server/server.go

```diff
@@ -482,3 +474,36 @@ func (f *filteredWriter) Write(p []byte) (n int, err error) {
 	}
 	return len(p), nil
 }
+
+func GetTLSConfig(tlsConfig *TLSConfig) (TLSConfig *tls.Config, err error) {
```

Comment:
Minor: I would either log.Fatal all the errors in this function or just return wrapped errors, they seem to be to be roughly equivalent in severity

Reply:
Good catch. This code moved from somewhere else and I forgot to change it to return the error.